### DISCUSSION TITLE
How to disable motion detection?

### DISCUSSION THREAD

**Discussion by strips** [Motion-Project/motionplus#37] [Q&A] [answered]
Hi,

How do I disable motion detection?
I cant seem to find this in the docs or config files.

Regards
Stian

**Comment by gabbymouth** [comment, marked as answer]
I do it for a camera via the command line like this: curl -X POST
http://192.168.x.xxx:8080 -d "camid=###&command=pause"

Or you can use the GUI and select the Action Pause.

**Reply by strips** [reply]
Thanks :-D

Some times I'm just blind as a mole. Kind of self explainatory really :)
